mlir/include/MBQC/IR/MBQCOps.td

```diff
@@ -30,8 +30,8 @@ def MeasurementPlane : I32EnumAttr<"MeasurementPlane",
     "Planes in the Bloch sphere representation with support for arbitrary-basis measurements",
     [
         I32EnumAttrCase<"XY", 0>,
-        I32EnumAttrCase<"ZX", 1>,
-        I32EnumAttrCase<"YZ", 2>,
+        I32EnumAttrCase<"YZ", 1>,
+        I32EnumAttrCase<"ZX", 2>,
```

Comment:
Just wondering if there's any ambiguity here with how we define the planes in the PennyLane implementation? Do we match the semantics/ordering of the planar indices?

Update: just checked, all good :+1: